real_time/arduino_src/radio_buggy_mega/main.cpp

```diff
@@ -46,8 +48,8 @@
     #define PWM_SCALE_STORED_ANGLE 1000 // in hundredths of a degree for precision
     #define POT_OFFSET_STEERING_IN 121
     #define POT_SCALE_STEERING_IN -10
-    #define STEERING_LIMIT_LEFT -1000
-    #define STEERING_LIMIT_RIGHT 1000
+    #define STEERING_LIMIT_LEFT -1500   //steering_set assumes that left is less than right.
+    #define STEERING_LIMIT_RIGHT 1500
```

Comment:
That's nice? Left has been negative one the buggy for the last 2 years. Neither answer is right, but consistency is important.